students/AndrewMiotke/session02/series.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
```

Comment:
This looks great!

Reply:
Awesome, thank you! I didn't think it fulfilled all of requirements @nairraghav